config/migrations/2023/20230504192246-add-mandataris-status-code-effectief.sparql

```diff
@@ -0,0 +1,16 @@
+PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/> 
+
+# Add status "EFFECTIEF" to erediest mandatarissen & ministers if they dont already have a status assigned
+INSERT {
+  GRAPH ?h {
+     ?mandataris <http://data.vlaanderen.be/ns/mandaat#status> <http://data.vlaanderen.be/id/concept/MandatarisStatusCode/21063a5b-912c-4241-841c-cc7fb3c73e75> #Effectief
+  }
+} WHERE {
+  GRAPH ?g {
+    ?mandataris a ere:EredienstMandataris
+    FILTER NOT EXISTS {
+      ?mandataris <http://data.vlaanderen.be/ns/mandaat#status> ?statusCode
+    }
+  }
+  BIND(?g as ?h)
```

Comment:
Good question. The mandatarissen live in their own organization graphs. I cant just hardcode the graph in the query otherwise I need to individually create a query for each graph I am manipulating. The bind makes sure that the graph where the mandataris triple lives is where we insert the new mandataris-status-code triple. 

Bind ?g as ?h

Bind graph where we found the mandataris (?g) as the graphs where we want to insert the new status triple (?h)

Reply:
Would it have been the same had you used `GRAPH ?g` in `INSERT` and `GRAPH ?g` in `WHERE`?

Reply:
Yeah thats the same, bind is a bit more explicit